### PR TITLE
[FLINK-29878][hive] Fix flink-sql-connector hive error

### PR DESCRIPTION
**Pull request by Samrat002**
## Brief change log
Fix dependencies error 

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers:  no 
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no 
  - The S3 file system connector:  no 

## Documentation

  - Does this pull request introduce a new feature?  no

### CODE REVIEW

**Comment by Samrat002** [comment]
@flinkbot run azure

File: flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml [hunk 1]
```diff
@@ -151,8 +157,6 @@ under the License.
 										<exclude>avro/shaded/com/google/**</exclude>
 										<exclude>org/apache/avro/**</exclude>
 										<exclude>META-INF/maven/org.apache.avro/**</exclude>
-										<exclude>org/apache/hadoop/hive/conf/HiveConf*</exclude>
```
Comment (luoyuxia) [inline flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml:154]: We can use that just as before:
```
<exclude>org/apache/hadoop/hive/conf/HiveConf.class</exclude>							<exclude>org/apache/hadoop/hive/metastore/HiveMetaStoreClient.class</exclude>
```

**Comment by Samrat002** [comment]
```
Nov 07 19:06:18 java.lang.AssertionError: Create test topic : partition_failure_recovery_EventTime failed, org.apache.kafka.common.errors.TopicExistsException: Topic 'partition_failure_recovery_EventTime' already exists.
Nov 07 19:06:18 	at org.apache.flink.streaming.connectors.kafka.KafkaTestEnvironmentImpl.createTestTopic(KafkaTestEnvironmentImpl.java:207)
Nov 07 19:06:18 	at org.apache.flink.streaming.connectors.kafka.KafkaTestEnvironment.createTestTopic(KafkaTestEnvironment.java:97)
Nov 07 19:06:18 	at org.apache.flink.streaming.connectors.kafka.KafkaTestBase.createTestTopic(KafkaTestBase.java:217)
Nov 07 19:06:18 	at org.apache.flink.streaming.connectors.kafka.shuffle.KafkaShuffleExactlyOnceITCase.testAssignedToPartitionFailureRecovery(KafkaShuffleExactlyOnceITCase.java:158)
Nov 07 19:06:18 	at org.apache.flink.streaming.connectors.kafka.shuffle.KafkaShuffleExactlyOnceITCase.testAssignedToPartitionFailureRecoveryEventTime(KafkaShuffleExactlyOnceITCase.java:101)
Nov 07 19:06:18 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```
not related test failure !

**Review by luoyuxia — APPROVED** [review]
@Samrat002 Thanks for fix. LGTM

**Comment by luoyuxia** [comment]
@flinkbot run azure

**Review by dannycranmer — APPROVED** [review]

**Comment by dannycranmer** [comment]
LGTM, thanks @Samrat002

**Pull request merged**